simpledotorg/simple-server
# Chartjs - DRY bar chart config and implement new config on all bar charts

**Story card:** [sc-9833](https://app.shortcut.com/simpledotorg/story/9833/dry-bar-charts-config)

## Because

Lots of duplicate configuration when constructing the charts on the dashboard

## This addresses

Reduce repetition of bar chart config by:

- Creating a base bar chart config
- Implement baseConfig for all bar charts (HTN and DM)
- Remove old unused config functions

## Test instructions

Head to the dashboard and check that all bar charts:
1. Load without error
2. Look the same as charts on the live server
3. Are interactive as expected
4. On hovering the relevant figures update or tooltip shows
